README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
i do not think we are communicating because i asked why you are trying to change the meaning of the sections only to be ignored. in a discussion or a conversation, "why" question is always followed by "because". the second question seems to get an answer, but is not. your answer does not explain why your are nitpicking.

despite the unclear answers, your strong opposition to portable roles has been understood. here is logical consequences from your opinion; why don't you oppose an idea to support platforms, as someone expressed in the past, that we do not use? that is just a waste of time. it is much easier to write non-portable code. we should bails out if the platform is not supported, or `uname -r` is not `Linux`, because we have not tested and running a role on unsupported platforms would kill kittens.

in short, you are suggesting new policy to be implemented, not pointing out defects, bugs, or anything in the role itself. I really appreciate if you propose elsewhere, not in this PR, because the PR is meant to solve issues in the development. if you want new policy, this is wrong place.

if you are okay with other part of the code, please approve. feel free to create an issue with your proposals. the PR is blocking other PR and the development. thank you for being productive.